gui/src/app/FileEditor/TextEditor.tsx

```diff
@@ -1,6 +1,9 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
+import { Hyperlink, SmallIconButton } from "@fi-sci/misc";
 import { Editor } from "@monaco-editor/react";
 import { Refresh, Save } from "@mui/icons-material";
+import { highlightJsData } from "@SpComponents/stanLang";
```

Comment:
This import coming from the Components alias strikes me as odd

Reply:
I can see that--just a byproduct of where the file is actually located within the project. It might make more sense (since this is defining a bunch of language keywords) to put it in `Stanc`?

Reply:
Maybe -- I want to re-vamp the highlighting eventually. Then it might make sense to have a 'stan language' folder with 'compiler' as a sub-folder or something. Nothing for now, I think.